### ISSUE TITLE
Quick-Menu not aligned on some sites

### ISSUE THREAD

**Issue by ThomasSeeker** [ssborbis/ContextSearch-web-ext#305]
Hello, ... I use ContextSearch every day ... and sometimes (but very rarely) it happens, that the Quick-Menu does not position itself in the vertical center of the screen (like usually) but instead appears below the highlighted word (even if this is at the bottom of the page).

Here is an example for such a site: 

http://www.f-nietzsche.de/lou3.htm

Is this a bug?

Best greetings!

**Comment by ssborbis** [comment]
How do you have the menu configured when opening? The default behaviour is to open relative to the mouse cursor, not in the vertical center of the screen. 

My config, for instance is:
![position](https://user-images.githubusercontent.com/24310997/104961726-85d48600-599c-11eb-82fe-88489ddfa655.png)

... when I highlight a word and open the menu it displays this
![opened](https://user-images.githubusercontent.com/24310997/104961763-9684fc00-599c-11eb-952c-249cb48fce49.png)

There are tons of menu configurations possible, so your setup may be different than mine, but if the menu has always opened in the vertical center of the page using default settings, there may be a conflict with another addon. Give me some more details about your setup.

**Comment by ThomasSeeker** [comment]
Ah, thank you for your reply ssborbis!  ... it lead me to a solution (workaround(?)).

I have set the menu to show text in rows. Having 22 rows in a certain size makes my Quick-Menu fill the whole height of the screen. 

I had set the position exactly like you do. But because its large size it showed always in the vertical center, so its full height could be seen (Like I wanted it to.) The horizontal position was right to the cursor, like it should be too. 

And my above post was about the rare case, when the menu vertically showed right below the cursor. When the cursor was at the bottom of the page, the menu was mostly hidden. This might be a bug?

But after your answer I found a workaround: Using the top right position. Now the full menu shows in the vertical center on these pages too!

Thank you for this wonderful addon! :-)
